fs-storage/tests/FileStorage.java

```diff
@@ -0,0 +1,69 @@
+public class FileStorage {
```

Comment:
Could you create the counter-part PR in [ark-android](https://github.com/ARK-Builders/ark-android) which uses this branch as backend for [the storage demo](https://github.com/ARK-Builders/ark-android/pull/97)?

Reply:
@twitu @Pushkarm029

Reply:
As we discussed this will be done in a separate PR, once the ark-rust repo is refactored to move all java related code into a single java project.